store/postgres/src/writable.rs

```diff
@@ -1217,6 +1228,14 @@ impl Queue {
         Ok(map)
     }
 
+    fn get_range(
+        &self,
+        entity_type: &EntityType,
+        block_range: Range<BlockNumber>,
+    ) -> Result<BTreeMap<BlockNumber, Vec<Entity>>, StoreError> {
+        self.store.get_range(entity_type, block_range)
```

Comment:
This will return the wrong result if `block_range` covers blocks that are still in the queue - those aren't in the database yet. You'd need to use the `BlockTracker` to go through the queue and find entities for those blocks. You also only want to look in the store for the part of `block_range` that isn't in the queue.

Reply:
I've removed it from the `Queue` and from the `Writer` and left if only in the `WritableStore` with additional comment. Would that be acceptable? Alternative is to create `ReaderStore` struct.

Reply:
I decided to go with a new trait since the methods of readStore would need to be implemented in WritableStore too.

Reply:
Right now the `WritableStore` is a private member of the `SourceableStore` struct. It is only used for implementing the `DeploymentCursorTracker` methods of the `SourceableStore` trait, which replaces ReadStore in this context. I hope that addresses your concerns sufficiently.

Reply:
Ok finally removed all traces of `WritableStore` from the `SourceableStore`. Feel free to review it with the other two PRs. Sorry for the confusion before.